scripts/get-release-version.js

```diff
@@ -71,6 +71,33 @@ export function getVersion(options = {}) {
       latestNightlyTag.replace(/-nightly.*/, '').replace(/^v/, '') + '-preview';
     npmTag = 'preview';
     previousReleaseTag = getLatestTag('contains("preview")');
+  } else if (type === 'patch') {
+    const patchFrom = options.patchFrom || args.patchFrom;
+    if (!patchFrom || (patchFrom !== 'stable' && patchFrom !== 'preview')) {
+      throw new Error(
+        'Patch type must be specified with --patch-from=stable or --patch-from=preview',
+      );
+    }
+
+    if (patchFrom === 'stable') {
+      previousReleaseTag = getLatestTag(
+        '(contains("nightly") or contains("preview")) | not',
+      );
+      const [major, minor, patch] = previousReleaseTag
+        .replace(/^v/, '')
+        .split('.');
+      releaseVersion = `${major}.${minor}.${parseInt(patch) + 1}`;
```

Comment:
> parseInt(undefined) === NaN
> NaN + 1 === NaN

I worry that if we `patch` is undefined (which seems potentially likely? Maybe I'm paranoid or just don't understand enough) -- you could end up with a release version of `1.6.NaN`.

Something like this is probably safer (or I'm over analyzing it?):

```
(Number.isFinite(parseInt(undefined)) || 0) + 1
```